exercises/binary_converter.cpp

```diff
@@ -5,3 +5,38 @@
   Insert first number: 8
   The binary number is: 1000
 */
+#include <stdio.h>
+#include <iostream>
+using namespace std;
+
+int* toBinary(int n){
+  int* binaryNumber = new int[20];
+  int i = 0;
+  while (n != 0){
+    binaryNumber[i] = n%2;
+    i++;
+    n /= 2;
+  }
+  return binaryNumber;
+
+}
```

Comment:
hai messo un ritorno a capo poco prima di chiudere la funzione, potresti usare le newline in maniera più distribuita nel codice (es. prima e dopo del while) per rendere il codice più leggibile